src/handlers/GraphQLHandler.ts

```diff
@@ -48,6 +50,7 @@ export const graphqlContext: GraphQLContext<any> = {
   extensions,
   errors,
   cookie,
+  field: field,
```

Comment:
You can use shorthand assignment here!

```suggestion
  field,
```

Reply:
✅